source/guide_headscale.rst

```diff
@@ -0,0 +1,214 @@
+.. author:: Lukas Wolfsteiner <https://lukas.wolfsteiner.media>
+
+.. tag:: vpn
+.. tag:: wireguard
+.. tag:: self-hosting
+.. tag:: lang-go
+.. tag:: web
+
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/headscale.svg
+      :align: center
+
+#########
+Headscale
+#########
+
+.. tag_list::
+
+Headscale_ is an open source, self-hosted implementation of the Tailscale control server. It allows you to create your own private mesh VPN using the WireGuard protocol, providing secure connectivity between your devices without relying on a third-party service.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`web backends <web-backends>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+Headscale is released under the _[BSD-3-Clause License](https://github.com/juanfont/headscale/blob/main/LICENSE)_.
+
+All relevant legal information can be found here:
+
+  * https://github.com/juanfont/headscale/blob/main/LICENSE
+
+Prerequisites
+=============
+
+We're using the latest headscale release for Linux x86_64. Check the official releases_ for the latest version.
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Download the latest headscale binary from the official GitHub releases page. Replace ``<VERSION>`` with the latest version (e.g., ``0.26.1``):
+
+::
+
+ [isabell@stardust ~]$ wget -O ~/bin/headscale \
+   https://github.com/juanfont/headscale/releases/download/v<VERSION>/headscale_<VERSION>_linux_amd64
+ [isabell@stardust ~]$ chmod +x ~/bin/headscale
+
+.. note:: Ensure ``~/bin`` is in your ``$PATH``. You can check with ``echo $PATH``.
+
+Create a directory for headscale data, configuration and logs:
+
+::
+
+ [isabell@stardust ~]$ mkdir -p ~/opt/headscale
+ [isabell@stardust ~]$ mkdir -p ~/etc/headscale
+ [isabell@stardust ~]$ mkdir -p ~/logs/headscale
+
+Download the example configuration file and adjust it:
+
+::
+
+ [isabell@stardust ~]$ wget -O ~/etc/headscale/config.yaml \
+   https://github.com/juanfont/headscale/raw/v<VERSION>/config-example.yaml
+ [isabell@stardust ~]$ nano ~/etc/headscale/config.yaml
+
+.. warning::
+  Review and adjust the configuration to suit your environment. At minimum, set the ``server_url``, ``private_key_path``, and ``db_type``/``db_path``. For a simple SQLite setup, use:
+
+  .. code-block:: yaml
+
+    server_url: "https://<your-domain>:443"
+    private_key_path: "~/etc/headscale/private.key"
+    db_type: "sqlite"
+    db_path: "~/opt/headscale/db.sqlite"
```

Comment:
Same as above, `db_path` is nowhere to be found, instead there's 

```
  # SQLite config
  sqlite:
    path: /var/lib/headscale/db.sqlite
```

Reply:
Resolved in https://github.com/Uberspace/lab/pull/1904/commits/18d9ef915790e26e6a4713b2633c1e23ca1a2bee.